### PR TITLE
feat(SearchQueryShortMessage): implement

### PR DESCRIPTION
**Pull request by TyaglovOV**

### CODE REVIEW

File: src/components/SidebarSearchResults/README.md [hunk 1]
```diff
@@ -2,41 +2,45 @@
...
+  },
 };
 
 const togglePending = () => {
   setState({
     messages: {
       ...state.messages,
-      pending: state.messages.pending ? false : true
-    }
+      pending: state.messages.pending ? false : true,
```
Comment (mnasyrov) [inline src/components/SidebarSearchResults/README.md]: Why not `!state.messages.pending`?

File: src/components/SidebarSearchResults/types.js [hunk 2]
```diff
@@ -19,4 +19,5 @@ export type SidebarSearchResultsProps = {
   messages: Field<SearchEntity[]>,
   onGoToPeer: (peer: Peer) => mixed,
   onGoToMessage: (peer: Peer, message: Message) => mixed,
+  necessaryQueryLength?: number,
```
Comment (mnasyrov) [inline src/components/SidebarSearchResults/types.js]: IMHO, `minQueryLength` sounds better.

File: src/components/SidebarSearchResults/README.md (hunk 1 above)
Comment (TyaglovOV) [reply src/components/SidebarSearchResults/README.md]: Coz i was tired and didn't see it :D

File: src/components/SidebarSearchResults/types.js (hunk 2 above)
Comment (TyaglovOV) [reply src/components/SidebarSearchResults/types.js]: Yep, naming was never my strong side.

File: src/components/SidebarSearchResults/README.md [hunk 3]
```diff
@@ -45,38 +49,85 @@ const toggleError = () => {
...
 <div>
   <div className="styleguide__buttons">
-    <Button onClick={togglePending} size="small" theme="primary" style={{ marginRight: 4 }}>Toggle Pending</Button>
-    <Button onClick={toggleError} size="small" theme="primary" style={{ marginRight: 4 }}>Toggle Error</Button>
-    <Button onClick={toggleResults} size="small" theme="primary" style={{ marginRight: 4 }}>Toggle Results</Button>
+    <Button
+      onClick={togglePending}
+      size="small"
+      theme="primary"
+      style={{ marginRight: 4 }}
```
Comment (meafmira) [inline src/components/SidebarSearchResults/README.md]: Why use `style` four times? Replace with class

File: src/components/SidebarSearchResults/README.md [hunk 4]
```diff
@@ -45,38 +49,85 @@ const toggleError = () => {
...
+    }}
+  >
     <SidebarSearchResults
       query={state.query}
       peers={state.peers}
       messages={state.messages}
+      minQueryLength={3}
       onGoToPeer={(peer) => console.debug('Go to peer', peer)}
-      onGoToMessage={(peer, message) => console.debug('Go to message', peer, message)}
+      onGoToMessage={(peer, message) =>
```
Comment (meafmira) [inline src/components/SidebarSearchResults/README.md]: Remove logging

File: src/components/SidebarSearchResults/README.md (hunk 3 above)
Comment (TyaglovOV) [reply src/components/SidebarSearchResults/README.md]: done

File: src/components/SidebarSearchResults/README.md (hunk 4 above)
Comment (TyaglovOV) [reply src/components/SidebarSearchResults/README.md]: done

**Pull request merged**